ark/spLDA/processing.py

```diff
@@ -1,86 +1,186 @@
 import functools
-import matplotlib.pyplot as plt
+
 import numpy as np
-import pandas as pd
-import os
-import pickle
-from ark.utils.spatial_lda_utils import check_format_cell_table_args
-# from spatial_lda.featurization import *
-# from spatial_lda.visualization import plot_adjacency_graph()
+from sklearn.model_selection import train_test_split
+from spatial_lda import featurization as ft
 
+from ark.settings import BASE_COLS
+from ark.utils.spatial_lda_utils import check_format_cell_table_args, \
+    check_featurize_cell_table_args
```

Comment:
If you don't expect `spatial_lda_utils` to become extremely massive, I recommend just importing the entire `ark.utils.spatial_lda_utils` module. Generally, I try not to use line continuations for import statements.

Another option is to break this into two separate imports or to use shorter function names.